torchgen/gen_lazy_tensor.py

```diff
@@ -199,7 +199,11 @@ class default_args:
     shape_inference_hdr: str = "torch/csrc/lazy/core/shape_inference.h"
     tensor_class: str = "torch::lazy::LazyTensor"
     tensor_class_hdr: str = "torch/csrc/lazy/core/tensor.h"
+    shape_class: str = "torch::lazy::Shape"
```

Comment:
It doesn't seem like we actually use this in XLA.

Reply:
Removed! This was a part of an older commit, but now it isn't needed since PyTorch/XLA overrides the entire `gen()` function. Good catch!